lib/ServiceManager.ts

```diff
@@ -43,14 +44,15 @@ export class ServiceManager implements ServiceManagerInterface {
 
     // TODO: backwards compatibility, remove in next major version - OKTA-473815
     const { autoRenew, autoRemove, syncStorage } = sdk.tokenManager.getOptions();
+    options.electionChannelName = options.broadcastChannelName;
```

Comment:
this should be:

```
options.electionChannelName = options.electionChannelName || options.broadcastChannelName
```

to support passing `electionChannelName` (as is being done in the test app)